tools/wptrunner/wptrunner/browsers/webkitgtk_minibrowser.py

```diff
@@ -23,17 +23,39 @@ def check_args(**kwargs):
     pass
 
 
+def maybe_add_args(required_args, current_args):
```

Comment:
Since the hack is used in two products, can this helper be moved into a base class?